src/components/organisms/Home.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+	<div class="home">
+		<HomeImageContainer v-bind:imageContainer="imageContainer" v-bind:key="imageContainer.id" v-for="imageContainer in imageContainers"/>
+		<HomeArrowContainer v-on:traverse="switchImage" v-bind:arrows="arrowContainer"/>
+	</div>
+</template>
+
+<script>
+	import HomeArrowContainer from "../molecules/HomeArrowContainer.vue"
+	import HomeImageContainer from "../molecules/HomeImageContainer.vue"
+	import json from "../../../data/structure.json"
+	export default {
+		name: "home",
+		components: {
+			HomeArrowContainer,
+			HomeImageContainer
+		},
+		methods: {
+			switchImage(direction) {
+				const imagesInCarousel = this.imageContainers.length;
+				let active = 0;
+				for (; active < imagesInCarousel-1 ; active++) {
```

Comment:
use forEach instead of for, it has much neater syntax and is way more readable it is 2021 not 2010